src/bp/core/module-loader.ts

```diff
@@ -167,6 +176,16 @@ export class ModuleLoader {
     return true
   }
 
+  public async modulesMigrationRequest(dryRun?: boolean) {
+    const modules = this.getLoadedModules()
+
+    return Promise.map(modules, async module => {
```

Comment:
`mapSeries` would be safer IMO when doing migration work, so that we only alter things sequentially.